XBVR crashes if videos are played to the end via DeoVR

Sometimes XBVR crashes if I let the video play to the end. This does not happen if I end the video earlier.

Here's the log:

```
[454078]  INFO Session #15528 duration for scene #6223 is 488.706044407
[454078]  INFO New session #15529 for scene #3590 from deovr
[454559]  INFO Session #15529 duration for scene #3590 is 481.083851922
[454571]  INFO New session #15530 for scene #6026 from file
[454661]  INFO Session #15530 duration for scene #6026 is 90.529203263
[454664]  INFO New session #15531 for scene #5929 from file
[454678]  INFO Session #15531 duration for scene #5929 is 13.876269287
[454678]  INFO New session #15532 for scene #1878 from deovr
panic: runtime error: index out of range

goroutine 5582605 [running]:
github.com/xbapps/xbvr/pkg/session.TrackSessionFromRemote(0xc003e72340, 0x34, 0x4081711eb851eb85, 0x40817015182a9931, 0x3ff0000000000000, 0x0)
        /go/appsrc/pkg/session/session.go:96 +0x3c5
created by github.com/xbapps/xbvr/pkg/session.deoLoop
        /go/appsrc/pkg/session/remote.go:77 +0x9a4
```

--- a/pkg/session/remote.go
+++ b/pkg/session/remote.go
@@ -20,6 +20,7 @@ type DeoPacket struct {
 
 const PLAYING = 0
 const PAUSED = 1
+const FINISHED = 2
 
 var DeoPlayerHost = ""
 var DeoRequestHost = ""

--- a/pkg/session/session.go
+++ b/pkg/session/session.go
@@ -96,7 +96,7 @@ func TrackSessionFromRemote(packet DeoPacket) {
 		lastSessionEnd = time.Now()
 
 		position := int(packet.CurrentTime)
-		if position != 0 && len(currentSessionHeatmap) >= position {
+		if position > 0 && position < len(currentSessionHeatmap) {
 			currentSessionHeatmap[position] = currentSessionHeatmap[position] + 1
 		}
 	}

